src/content/docs/react-components/accordion.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: Accordion
```

Comment:
Please create one more tab where you need to specify the additional libraries that you have used. For example: **import { motion, AnimatePresence } from 'framer-motion';**

So please create a one more tab where you need to specify these libraries and have it as first tab.

Follow this order:

Installation of Libraries => Usage => Code of component.

Reply:
Please follow this for all other components as well in case if you have used additional libraries for these components to work.